runtime_builders/publish_runtime.py

```diff
@@ -0,0 +1,166 @@
+#!/usr/bin/python
+
+# Copyright 2016 Google Inc. All rights reserved.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+from datetime import datetime
+import json
+import logging
+import os
+import regex
+from ruamel import yaml
+import subprocess
+import sys
+import tempfile
+
+from google.cloud import storage
+
+LANGUAGES = [
```

Comment:
yeah, I really only did this for naming purposes (since the files that are passed in will all be called "cloudbuild.yaml.in" or something generic). gonna go ahead and remove this though in favor of passing a required "runtime-name" parameter.